server/cli/src/plugins.rs

```diff
@@ -110,7 +113,7 @@ struct ManifestJson {
     om_supply_plugin: PluginDescription,
 }
 
-pub(crate) fn generate_plugin_bundle(
+pub fn generate_plugin_bundle(
```

Comment:
just one method to yarn install and convert_data, it's actually faster now without using --check-files and using lock files (but not comiting lock file)